src/atoms/Text/index.js

```diff
@@ -5,19 +5,85 @@ import classnames from 'classnames';
 import colors from 'atoms/Color/colors.module.scss';
 import styles from './text.module.scss';
 
+
+function Em(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'em',
+  });
+
+  return Text(newProps);
+}
+
+function Strong(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'strong',
+  });
+
+  return Text(newProps);
+}
+
+function Strike(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'span',
+    variant: 'strikethrough'
+  });
+
+  return Text(newProps);
+}
+
+function Sup(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'sup',
+  });
+
+  return Text(newProps);
+}
+
+const getTag = (props) => {
+  const {
+    tag,
+    semibold,
+    variant,
+  } = props;
+
+  if (semibold) return 'strong';
+  if (variant === 'italics') return 'em';
+  if (variant === 'strikethrough') return 'span';
```

Comment:
Should we make the `italics` and `strikethrough` bools as well, since we have these neat "get" functions to figure things out?